### PR TITLE
upgrade to Jekyll 3 beta

### PR DESCRIPTION
**Pull request by afeld**
You know, just for funsies :smile:

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,8 +13,10 @@ gem 'uglifier'
 
 group :jekyll_plugins do
   gem 'jekyll-assets'
-  gem 'jekyll_pages_api'
-  gem 'liquid_pluralize'
+  # https://github.com/18F/jekyll_pages_api/pull/21
+  gem 'jekyll_pages_api', git: 'https://github.com/18F/jekyll_pages_api.git', ref: '608affe083e15ef73ac1317e43c7a3f78006630e'
```
Comment (mbland) [inline Gemfile]: I just merged 18F/jekyll_pages_api#21. If you cut a new gem, this part can go back to the way it was.

**Comment by mbland** [comment]
I'm curious: Do you notice any speedup with 3.0.0.beta2? I couldn't with 3.0.0.beta1. Still, I'd be up for hopping on the bleeding edge once my comments are addressed.

File: Gemfile [hunk 2]
```diff
@@ -12,8 +13,10 @@ gem 'uglifier'
...
+  # https://github.com/bdesham/pluralize/pull/3
+  gem 'liquid_pluralize', git: 'https://github.com/bdesham/pluralize', ref: '057ffe2a174b7b5c084fb0d15f20dadcf0bf0fd9'
```
Comment (mbland) [inline Gemfile]: Any chance this gem'll get cut soon, too?

Comment (afeld) [inline Gemfile]: Asked here: https://github.com/bdesham/pluralize/pull/3#issuecomment-77573221

**Comment by afeld** [comment]
>  Do you notice any speedup with 3.0.0.beta2?

Basically the same :confused: 

| | Jekyll 2 | Jekyll 3 |
| --- | --- | --- |
| `rm -rf _site* && ./go build` | 13.27s user 1.84s system 103% cpu 14.632 total | 14.15s user 1.42s system 105% cpu 14.825 total |
| Regenerating after modifying `_layouts/bare.html` | 6.12s | 6.31s |

**Comment by afeld** [comment]
Updated to use the released versions of the gems :guitar:

**Comment by mbland** [comment]
Interesting that the "guitar" emoji is an EVH Wolfgang instead of a Strat or a Les Paul.

OK, here goes, why not...

**Comment by afeld** [comment]
#guitarnerd

**Comment by parkr** [comment]
Pipe dream: create a Liquid profiler.

Actual thing we can do now: run stackprof against a site and see what is taking so long.

**Pull request merged**